pkg/disk/partition_table.go

```diff
@@ -1428,7 +1428,14 @@ func addPlainPartition(pt *PartitionTable, partition blueprint.PartitionCustomiz
 		case partition.Mountpoint == "/boot":
 			typeName = "boot"
 		case fstype == "none":
-			typeName = "data"
+			switch partition.Label {
+			case "ukiboot_a", "ukiboot_b":
+				typeName = "ukiboot"
+			case "ukibootctl":
+				typeName = "ukibootctl"
```

Comment:
Nitpick: I'm still wondering if it wouldn't make sense to allow specifying the `partition.PartType` explicitly in the FS customization, instead of having a special case here based on the partition label.

IOW, instead of adding `plain` partition, handle new types of partitions -> `ukiboot` and `ukibootctl`.

It seems to me that trying to reuse the "plain" partition code for UKI partitions just adds "magic behavior".

Reply:
Yeah, I think this is what I tried to express in my previous comments as well (just less eloquently :)

Reply:
@thozza @mvo5 I see. I think that might work and makes sense. I believe that we should still introduce the UKIBootPartitionUUID and UKIBootCtlPartitionUUID constants though to make it easier for people to write the customizations without looking up what the GUID is for these partitions types.

But I agree that we can drop this mapping logic indeed and be explicit in the customization.

Reply:
@thozza @mvo5 thanks a lot folks for your suggestion. It makes a lot of sense to me and it indeed makes things more simple and avoids special logic to handle the UKIBoot partition types.

Can you please tell me what you think of latest version that I pushed ?